### PR TITLE
google-cpi: properties as of v25.10.0

### PR DESCRIPTION
**Pull request by johnsonj**
Changed the format for this to sync with the CPI documentation. I think it's easier for the viewers and easier for us to keep the two in sync. 

fixes cloudfoundry-incubator/bosh-google-cpi-release/issues/233

### CODE REVIEW

File: google-cpi.html.md.erb [hunk 1]
```diff
@@ -75,17 +78,25 @@ networks:
...
+| `ram`                   | Y        | Integer                                  | `2048`                                                                         | Amount of memory ([Google Compute Engine Custom Machine Types](https://cloud.google.com/custom-machine-types/)) the CPI will use when creating the instance (required if not using `machine_type`)
+| `zone`                  | N        | String                                   | `us-west1-a`                                                                   | The name of the [Google Compute Engine Zone](https://cloud.google.com/compute/docs/zones) where the instance must be created
+| `root_disk_size_gb`     | N        | Integer                                  | `10`                                                                           | The size (in Gb) of the instance root disk (default is `10Gb`)
```
Comment (dpb587-pivotal) [inline google-cpi.html.md.erb]: A default of `10Gb` would suggest to me that I need to include `Gb` suffix in the value, although I think it's supposed to be just the number.

**Comment by dpb587-pivotal** [comment]
Fixes #398

**Comment by johnsonj** [comment]
To be transparent, I went with this format to match our CPI docs. I did merge the information on bosh.io (added the defaults column). 

IMO it's easier to use this information as a table now that the lists are quite long. I'd vote for all the CPIs moving to this format or coming up with something better. 

If that's a non starter I can roll this back and just add the few fields, but I am concerned about keeping the two lists in sync and don't want to adopt the list format in our docs.

**Comment by cppforlife** [comment]
im not sure tables deal very well with nested properties (dont think google cpi has them though).

why do we maintain two set of docs? can we kill github ones?

**Comment by johnsonj** [comment]
Is there a way we can embed docs from the cpi to bosh.io? Link would be so-so.

What I like about having the docs in the same project as the CPI is we can ensure as we add new features they are merged with docs. It also means the docs in the repo at a given tag represent the feature set. Separate repos complicate this.

**Comment by dpb587-pivotal** [comment]
I strongly vote for the CPI repository to be the authoritative docs, and bosh.io should be able to embed those. Not the other way around. I agree with @johnsonj.

If consistent docs across CPIs would be an issue, there's always the schema option for a more structured form of documenting supported properties.

**Comment by cppforlife** [comment]
hmm i could get on board with that. need to figure out though how to keep
it all consistent across different cpi repos.

**Comment by dpb587-pivotal** [comment]
For the sake of getting docs updated to list the latest properties, I converted the table to the currently-typical list format. I autogenerated it from the bosh-google-cpi-release docs to retain consistency and make it predictable in the future. Eventually, I hope we can embed docs from the CPI release itself and avoid this sort of translation issue.

**Comment by johnsonj** [comment]
Nice extraction code! thanks

**Pull request merged**